api/users.go

```diff
@@ -117,9 +118,10 @@ func SearchUser(c *gin.Context) {
 }
 
 type userForLecturerDto struct {
-	ID    uint   `json:"id,omitempty"`
-	Name  string `json:"name,omitempty"`
-	Login string `json:"login,omitempty"`
+	ID       uint    `json:"id,omitempty"`
+	Name     string  `json:"name,omitempty"`
```

Comment:
Maybe we can call this `FirstName` to clear things up? This would then match the `firstName` in the `LdapResp`

Reply:
That would probably clear things up :+1: but I avoid migrating the database wherever I can -.- :smile:

Reply:
Ah, if this can cause issues, we better leave it as it is.

Reply:
It doesn't really cause issues, but it sure is annoying :D 
That's on me though, for not thinking forward to this